src/r2mm/installing/default_installation_rules/InstallationRuleApplicator.ts

```diff
@@ -1,158 +1,13 @@
-            buildBepInExRules("WestofLoathing"),
-            buildBepInExRules("SunHaven"),
-            buildBepInExRules("Wildfrost"),
-            buildBepInExRules("ShadowsofDoubt"),
-            buildBepInExRules("WeLoveKatamariRerollRoyalReverie"),
-            buildBepInExRules("Thronefall"),
-            buildBepInExRules("Techtonica"),
-            buildBepInExRules("GarfieldKartFuriousRacing"),
-            buildBepInExRules("WizardOfLegend"),
-            buildBepInExRules("BombRushCyberfunk"),
-            buildBepInExRules("TouhouLostBranchOfLegend"),
-            buildBepInExRules("WizardWithAGun"),
-            buildBepInExRules("Sunkenland"),
-            buildBepInExRules("Atomicrops"),
-            buildBepInExRules("Erenshor"),
-            buildBepInExRules("LastTrainOuttaWormtown"),
-            buildBepInExRules("Dredge"),
-            buildBepInExRules("CitiesSkylines2"),
-            buildBepInExRules("LethalCompany"),
-            buildBepInExRules("MeepleStation"),
-            buildBepInExRules("VoidCrew"),
-            buildBepInExRules("Sailwind"),
-            buildBepInExRules("Plasma"),
-            buildBepInExRules("ContentWarning"),
-            buildBepInExRules("BoplBattle"),
-            buildBepInExRules("Vertigo2"),
-            buildBepInExRules("AgainstTheStorm"),
-            buildBepInExRules("Lycans"),
-            buildBepInExRules("CastleStory"),
-            buildBepInExRules("Magicraft"),
-            buildBepInExRules("AnotherCrabsTreasure"),
-            buildBepInExRules("GladioMori"),
-            buildBepInExRules("SlipstreamRogueSpace"),
-            buildBepInExRules("BacktotheDawn"),
-            buildBepInExRules("BelowTheStone"),
-            buildBepInExRules("Gloomwood"),
-            buildBepInExRules("AmongUs"),
-            buildBepInExRules("BetrayalBeach"),
-            buildBepInExRules("ArcusChroma"),
-            buildBepInExRules("DeepRockGalacticSurvivor"),
-            buildBepInExRules("AleAndTaleTavern"),
-            buildBepInExRules("ScrewDrivers"),
-            buildBepInExRules("NineSols"),
-            buildBepInExRules("GoodbyeVolcanoHigh"),
-            buildBepInExRules("SupermarketTogether"),
-            buildBepInExRules("Shapez2"),
-            buildBepInExRules("PaqueretteDownTheBunburrows"),
-            buildBepInExRules("HardTime3"),
-            buildBepInExRules("TankTeam"),
-            buildBepInExRules("Distance"),
-            buildBepInExRules("FiveNightsAtFreddysIntoThePit"),
-            buildBepInExRules("GoreBox"),
-            buildBepInExRules("TCGCardShopSimulator"),
-            buildBepInExRules("OldMarketSimulator"),
-            buildBepInExRules("Subterranauts"),
-            buildBepInExRules("SULFUR"),
-        ]
+        InstallationRules.RULES = EcosystemSchema.r2mm_games
+            .map((x) => <CoreRuleType>{
+                gameName: x.r2modman!.internalFolderName,
+                rules: x.r2modman!.installRules,
```

Comment:
`!`s seem unnecessary

Reply:
Again, interesting, as for me removing the `!` doesn't cause an error, neither on IDE or when doing `yarn run run`. What version of TypeScript does the project install for you?